scaffold/Taskfile.yml

```diff
@@ -62,6 +63,26 @@ tasks:
     cmds:
       - task: drupal:update
 
+  tugboat:sync:
+    desc: "Syncs the database from production"
+    cmds:
+      - task: sync
+
+  tugboat:build:
+    desc: "Builds the project for Tugboat"
+    cmds:
+      - task: build
+
+  tugboat:update:
+    desc: "Updates the project for Tugboat"
+    cmds:
+      - task: update
```

Comment:
This is confusing and goes beyond the scope of getting Tugboat URLs working with Drush ULI.
Do we really need to address this in this pull request?